src/components/SignUp/OffstageRoles.tsx

```diff
@@ -1,22 +1,293 @@
+      sectionStateName: 'offstageRolesLighting',
+      textHeader: 'Lighting',
+      checkboxes: [
+        {
+          label: 'Lighting Design',
+          value: 'Lighting Design'
+        },
+        {
+          label: 'Projection Design',
+          value: 'Projection Design'
+        },
+        {
+          label: 'Special Effect Design',
+          value: 'Special Effect Design'
+        },
+        {
+          label: 'Master Electrician',
+          value: 'Master Electrician'
+        }
+      ]
+    },
+    sound: {
+      sectionStateValue: offstageRolesSound,
+      sectionStateName: 'offstageRolesSound',
+      textHeader: 'Sound',
+      checkboxes: [
+        {
+          label: 'Sound Design',
+          value: 'Sound Design'
+        },
+        {
+          label: 'Sound Mixer / Engineer',
+          value: 'Sound Mixer / Engineer'
+        }
+      ]
+    },
+    hairMakeupCostumes: {
+      sectionStateValue: offstageRolesHairMakeupCostumes,
+      sectionStateName: 'offstageRolesHairMakeupCostumes',
+      textHeader: 'Hair, Makeup, Costumes',
+      checkboxes: [
+        {
+          label: 'Costume Design',
+          value: 'Costume Design'
+        },
+        {
+          label: 'Hair & Wig Design',
+          value: 'Hair & Wig Design'
+        },
+        {
+          label: 'Make-up Design',
+          value: 'Make-up Design'
+        }
+      ]
+    }
+  };
+
+  const isRoleInRolesSection = (section: string[], role: string) =>
+    section.indexOf(role) > -1;
+
+  const rolesSectionChange = (
+    chkValue: any,
+    chk: string,
+    fieldName: string,
+    section: string[]
+  ) => {
+    let sectionRoles = [...section];
+    let name = fieldName;
+
+    if (chkValue) {
+      if (sectionRoles.indexOf(chk) < 0) {
+        sectionRoles.push(chk);
+      }
+    } else {
+      sectionRoles = sectionRoles.filter(rG => rG !== chk);
+    }
+
+    const target = {
+      name: 'offstageRolesGeneral',
+      value: sectionRoles
+    };
+
+    setForm({ target });
+  };
+
   return (
     <Container>
       <Row>
         <Col lg="12">
-          <h1>So, What Do You Like Doing?</h1>
-          <h2>Tell us what positions suit you best</h2>
+          <Title>SO, WHAT DO YOU LIKE DOING?</Title>
+          <TitleSecond>Tell us what positions suit you best.</TitleSecond>
+          <TitleThird>Off-Stage Roles</TitleThird>
+          <TitleFourth>Select all applicable positions</TitleFourth>
         </Col>
       </Row>
+
+      {Object.keys(offstageRolesObj).map(objKey => {
+        const currObjKey = (offstageRolesObj as any)[objKey as any];
+        const {
+          sectionStateValue,
+          sectionStateName,
+          textHeader,
+          checkboxes
+        } = currObjKey;
+
+        return (
+          <Row>
+            <Col lg="10">
+              <GenericAccordion
+                key={`accordion-${textHeader}`}
+                textHeader={textHeader}
+              >
+                <Col lg="6">
```

Comment:
You have another Col component outside of a Row component here. Col can only be used as a first-level child of Row, not nested in another Col.